sympy/printing/tests/test_octave.py

```diff
@@ -38,6 +38,7 @@ def test_Function():
     assert mcode(sin(x) ** cos(x)) == "sin(x).^cos(x)"
     assert mcode(abs(x)) == "abs(x)"
     assert mcode(ceiling(x)) == "ceil(x)"
+    assert mcode(Max(x, y) + Min(x, y)) == "max(x, y) + min(x, y)"
```

Comment:
You should add a test for more than 2 arguments too (see e.g. `test_ccode.py`). Probably worth doing for `fcode` as well (but not as critical since min and max in fortran are variadic).

Reply:
So the test I had in mind would be something along the lines of `mcode(Max(x, y, z)) == "max(x, max(y, z))"`

Reply:
Any other tests? @bjodah

Reply:
Looks good to me! Well done 👍